staging/src/k8s.io/apiserver/pkg/server/options/serving_test.go

```diff
@@ -401,6 +407,11 @@ func TestServerRunWithSNI(t *testing.T) {
 
 NextTest:
 	for title, test := range tests {
+		// this test needs tons of entropy. So better warn if it is low, making debugging of timeout much easier.
```

Comment:
certs/keys should be pre-generated... what needs entropy?

Reply:
now that you mention it I remember that we switched to pre-generated keys some time ago for the reason that it can take long.